eva_cttv_pipeline/evidence_string_generation/clinvar_to_evidence_strings.py

```diff
@@ -161,15 +165,18 @@ def clinvar_to_evidence_strings(string_to_efo_mappings, variant_to_gene_mappings
 
         # Failure mode 5 (skip). A ClinVar records has at least one trait with at least one valid name, but no suitable
         # EFO mappings were found in the database.
-        if not grouped_diseases:
+        if not grouped_diseases or not any(group[-1] for group in grouped_diseases):
             report.clinvar_skip_missing_efo_mapping += 1
             unmapped_trait_name = clinvar_record.traits_with_valid_names[0].preferred_or_other_valid_name
             report.unmapped_trait_names[unmapped_trait_name] += 1
+        # Only when grouped_diseases is completely absent are we unable to generate even unmapped evidence strings.
+        if not grouped_diseases:
```

Comment:
I have the feeling that `grouped_diseases` can never be empty anymore. 
Am I missing something ?
Also I don't understand the comment.

Reply:
I think you're right, I was thinking the record might have no traits with valid names, but at this point those have already been filtered out.